Update replace.asciidoc (#25055)
elastic/beats: libbeat/processors/actions/docs/replace.asciidoc

`replacement: replacement-string`, where:

* `field` is the original field name
* `pattern` is the regex pattern to match the field's value
* `replacement` is the replacement string to use to update the field's value

The `replace` processor cannot be used to replace value with a completely new value. 

> TIP: The `replacement` field value can be used to truncate the `field` value or replace

Reviewer: ```suggestion
TIP: The `replacement` field value can be used to truncate the `field` value or replace
it with a new string. It can also be used for masking PII information.
```

`replacement: replacement-string`, where:

* `field` is the original field name
* `pattern` is the regex pattern to match the field's value
* `replacement` is the replacement string to use to update the field's value

The `replace` processor cannot be used to replace value with a completely new value. 

TIP: Replace the field value to truncate part of the field value or replace 

> it with a new string. It can also be used for masking PII information.

Author: This reads really weird and I'm not quite sure what it should be saying. Replace the field value to truncate part of the value or replace it with a new string. It can also be used for masking PII information.

Is that better? Saying the field value twice didn't make sense.

Reviewer: Wow, yeah, this is a really tough one to decipher. Here's what I think it's trying to say:

```suggestion
TIP: The `replacement` field value can be used to truncate the `field` value or replace
```

The `replace` processor has following configuration settings:

> `ignore_missing`:: (Optional) If set to `true`, no error is logged in case a specified field 
> is missing. Default is `false`.

Reviewer: What about this?

```suggestion
`ignore_missing`:: (Optional) When `true`, no error is logged if the specified field 
is missing. Default is `false`.
```

> `fail_on_error`:: (Optional) If set to `true`, in case of an error the replacement of
> field values is stopped and the original event is returned. If set to `false`, replacement
> continues even if an error occurs during replacement. Default is `true`.

Reviewer: You made a good update to a similar config variable in another PR. I think it was something like this:

```suggestion
`fail_on_error`:: (Optional) If set to `true` and there's an error, the replacement of
field values is stopped and the original event is returned. If set to `false`, replacement
continues even if an error occurs during replacement. Default is `true`.
```
